### DISCUSSION TITLE
"Dangerous" link in issue template

### DISCUSSION THREAD

**Discussion by larsbarring** [SciTools/iris#4145] [General]
Hi,

I was just finishing up writing a bug report (*sorry for that!*) and then clicked on the link at the bottom 

> See [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) for further details.
</details>

to see if that was something for me. When going back to my surprise all (**all**) what I had entered was gone and I could not find a way (through Firefox history, etc. ) to get it back ....

Seems like this link should be surrounded by some suitable warning text.

Cheers
Lars

**Comment by trexfeathers** [comment]
I'm honestly not sure the link is necessary at all. It's a collapsible section, what more is there to say about it...

@bjlittle this was something you added; how would you feel about is removing the read more link?

**Comment by bjlittle** [comment]
@larsbarring Oops :blush:

Didn't quite realise it would nuke your content, apologies. I'll sort that for you and anyone else in the community that might trip over it.

Thanks for taking the time to report it here :+1: :beers:

**Comment by bjlittle** [comment]
See pull-request https://github.com/SciTools/iris/pull/4147

Fancy taking it on @trexfeathers ? :thinking:

**Reply by bjlittle** [reply]
@trexfeathers Awesome, thanks mate :tada:

**Comment by bjlittle** [comment]
Interesting... as this discussion has almost come to a conclusion, I was trying to work out how to close the discussion... but it seems that you can only **delete** or **lock** the discussion, which isn't what I want to do. Simply because, like closed issues and pull-requests, a historical archive is invaluable but I don't want resolved/answered discussions getting in the way of open/live discussions.

Scratching this itch, I swiftly stumbled across the following GitHub Discussion [github/feedback#3136](https://github.com/github/feedback/discussions/3136)... the irony of which wasn't lost on me :rofl: 

That discussion then led me to this connected GitHub Discussion [github/feedback#3097](https://github.com/github/feedback/discussions/3097) and in particular this [comment](https://github.com/github/feedback/discussions/3097#discussioncomment-294698), both of which I've :+1: to throw my weight behind the GitHub devs addressing this community issue, hopefully :pray:

**Comment by bjlittle** [comment]
@larsbarring Sorted :+1: 
![iris-issue-template](https://user-images.githubusercontent.com/2051656/118839710-9f3ffd00-b8be-11eb-8f14-6bfac9a763eb.png)
